blocks/api/registration.js

```diff
@@ -44,6 +49,19 @@ export function registerBlockType( name, settings ) {
 		);
 		return;
 	}
+	if ( ! settings ||
+			( settings && ! isFunction( settings.save ) ) ) {
+		console.error(
+			'A "save" function must be created within the block settings.'
+		);
+		return;
+	}
+	if ( settings && 'edit' in settings && ! isFunction( settings.edit ) ) {
```

Comment:
I guess the `settings` test is useless here, because we already checked for its existence above?